plugins/woocommerce/client/blocks/assets/js/base/components/cart-checkout/shipping-package-item-icon/index.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import type { CartItem } from '@woocommerce/types';
+
+/**
+ * Internal dependencies
+ */
+import type { PackageItem } from '../shipping-rates-control-package/types';
+import ProductImage from '../product-image';
+
+interface ProductImageProps {
```

Comment:
I feel `ShippingPackageItemIconProps` would be a better name as it more accurately reflects the component name.